cmd/protoc-gen-go-helpers/main.go

```diff
@@ -60,6 +60,7 @@ const (
 	msgImport  = "google.golang.org/protobuf/proto"
 	enumImport = "fmt"
 	headerTmpl = `
+// Code generated by protoc-gen-go-helpers. DO NOT EDIT.
```

Comment:
I meant in between the comment and the `package` line. I believe godoc interprets a comment on `package` as documentation for the package itself